Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -1012,11 +1015,12 @@ void RenderObject::repaintUsingContainer(SingleThreadWeakPtr<const RenderLayerMo
         repaintContainer = &view();
 
     if (CheckedPtr fragmentedFlow = dynamicDowncast<RenderFragmentedFlow>(*repaintContainer)) {
-        fragmentedFlow->repaintRectangleInFragments(r);
+        if (fragmentedFlow)
```

Comment:
This is redundant. line 1014/1017 already checks the nullity of `fragmentedFlow`.

Reply:
got it. Removing the redundant line.